OracleDatabase/RAC/OracleRealApplicationClusters/dockerfiles/18.3.0/crontabEntry

```diff
@@ -0,0 +1,11 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 1982-2019 Oracle and/or its affiliates. All rights reserved.
+#
+# Since: January, 2019
+# Author: sanjay.singh@oracle.com, paramdeep.saini@oracle.com
+# Description: Delete  a Grid node and add Oracle Database instance.
+#
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+
+* * * * * /tmp/resetFailedUnits.sh
```

Comment:
Can you explain why you're running this cronjob every minute of every day? Seems a bit excessive without context.

Reply:
Though I copied this file, it is not being used. It is for our internal customer (within Oracle) who wants to monitor failed units of systemd. I copied this file but not using it anywhere. Let me know if you want me to delete this file then I need to host this file internally  may be on orahub.

Reply:
If the file is not used, don't ship it.